src/lib/seam/components/AccessCodeTable/AccessCodeMenu.tsx

```diff
@@ -0,0 +1,115 @@
+import { CopyIcon } from 'lib/icons/Copy.js'
+import type { UseAccessCodeData } from 'lib/seam/access-codes/use-access-code.js'
+import { useDeleteAccessCode } from 'lib/seam/access-codes/use-delete-access-code.js'
+import { Button } from 'lib/ui/Button.js'
+import { copyToClipboard } from 'lib/ui/clipboard.js'
+import { MenuItem } from 'lib/ui/Menu/MenuItem.js'
+import { MoreActionsMenu } from 'lib/ui/Menu/MoreActionsMenu.js'
+import { useToggle } from 'lib/ui/use-toggle.js'
+
+export interface AccessCodeMenuProps {
+  accessCode: NonNullable<UseAccessCodeData>
+  onEdit: () => void
+  onViewDetails: () => void
+  disableEditAccessCode: boolean
+  disableDeleteAccessCode: boolean
+}
+
+export function AccessCodeMenu(props: AccessCodeMenuProps): JSX.Element {
+  return (
+    <MoreActionsMenu
+      menuProps={{
+        backgroundProps: {
+          className: 'seam-access-code-table-action-menu',
+        },
+      }}
+    >
+      <Content {...props} />
+    </MoreActionsMenu>
+  )
+}
+
+function Content({
+  accessCode,
+  onViewDetails,
+  disableEditAccessCode,
+  disableDeleteAccessCode,
+  onEdit,
+}: AccessCodeMenuProps): JSX.Element {
+  const [deleteConfirmationVisible, toggleDeleteConfirmation] = useToggle()
+
+  const deleteAccessCode = useDeleteAccessCode()
+
+  if (deleteConfirmationVisible) {
+    return (
+      <div className='seam-delete-confirmation'>
+        <span>Delete this code and data?</span>
```

Comment:
Move to `t.`

Reply:
updated in https://github.com/seamapi/react/pull/328/commits/3b08def1bc6db9e637ac008a38c0db44e07fb5aa 😃